railties/test/generators/migration_generator_test.rb

```diff
@@ -309,6 +309,16 @@ def test_add_migration_with_token_option
     end
   end
 
+  def test_add_migration_to_configured_path
+    old_paths = Rails.application.config.paths["db/migrate"]
+    Rails.application.config.paths.add "db/migrate", with: "db2/migrate"
+
+    migration = "migration_in_custom_path"
+    run_generator [migration]
+    Rails.application.config.paths["db/migrate"] = old_paths
```

Comment:
Same problem is in `model_generator_test`. I just mentioned that only once here. Sorry :(